authority-portal-frontend/src/app/common/components/form-elements/organization-roles-tooltip/organization-roles-tooltip.component.ts

```diff
@@ -19,11 +19,11 @@ import {Component} from '@angular/core';
 export class OrganizationRolesTooltipComponent {
   organizationRolesTooltip = `
 USER
-is the basic role that allows a person to open multiple participant related sections in portal but basically cannot change anything.
+Basic role that allows access to multiple participant related sections in the portal.
 
 KEY USER
-role allows creating and editing of organization's connectors and data offerings/requests.
+Can additionally manage their organization's connectors and data offerings/requests.
 
 ADMIN
-can additionally manage users, invite new users and update organization's profile.`;
+Can additionally manage their organization, e.g. change organization details and manage members.`;
```

Comment:
Either remove the periods here or add them in the application role tooltips to be more consistent. Personally, I would lean towards removing the periods

Reply:
done